_episodes/05-debug.md

```diff
@@ -0,0 +1,309 @@
+---
+title: "Debugging on ARCHER2"
+teaching: 30
+exercises: 30
```

Comment:
Time for the debugging exercise expanded and profiling moved to its own section to allow more time. This should close #24 .